<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>SARS-CoV-2 Variants of Concern in Switzerland</title>
    <style>
        body {
            max-width: 1000px;
            font-family: "Nunito Sans", sans-serif;
            font-weight: 400;
            margin-left: auto;
            margin-right: auto;
        }
        .source_info {
            padding-left: 20px;
            padding-right: 20px;
            font-size: small;
        }
        img {
            max-width: 100%;
        }
    </style>
</head>
<body>

<h1>SARS-CoV-2 Variants of Concern in Switzerland</h1>
<p>
    In a <a href="https://bsse.ethz.ch/cevo/research/sars-cov-2/swiss-sequencing-consortium---viollier.html">Swiss-wide genomic surveillance</a> effort, we currently
    perform whole genome sequencing of around 4-5% of all confirmed cases on a weekly basis (Viollier dataset below). These sequences are further tagged with a geographic location. 
    In addition, we assembled a dataset of a screening effort where the proposed B.1.1.7 variants are again confirmed through sequencing (Risch dataset below).
    Taken together, this leads to a characterization of around 11.5% of all confirmed cases since mid-December.
    
    We use this data to determine the increase in relative and absolute numbers of B.1.1.7. 
    In addition, for the whole genome sequencing data, we 
    provide results for Swiss regions and further track the variant 501Y.V2. We will add further Variants of Concern once they spread locally.
    We note that due the delay between infection and swab collection, the numbers for a particular day
    reflect the epidemic situation around 8-11 days earlier. 
    
   <!--- The methods employed for the plots below are described in a <a href="todo">pre-print</a> (not peer-reviewed). In particular, in this manuscript,
we additionally provide estimates on the transmission fitness advantage
    of B.1.1.7 for Switzerland and the Swiss regions. --->
    
    The code and data is available on
    <a href="https://github.com/cevo-public/Quantification-of-the-spread-of-a-SARS-CoV-2-variant">Github</a>.
</p>

<p><b>Important: the most recent estimates may be based on incomplete data and may thus be corrected later.</b></p>


<h2>B.1.1.7 (or S.501Y.V1) Variant</h2>

<p>First we show the estimated proportion of B.1.1.7 cases through time (vertical bars) together with the fitted logistic growth curve:</p>

<img
    src="https://raw.githubusercontent.com/cevo-public/Quantification-of-the-spread-of-a-SARS-CoV-2-variant/main/figures/g4.8/fig_proportion_switzerland_viollier_risch.svg"
    style="margin-left: 30px; margin-right: 30px;"
>

<img
    src="https://raw.githubusercontent.com/cevo-public/Quantification-of-the-spread-of-a-SARS-CoV-2-variant/main/figures/g4.8/fig_proportion_regions.svg"
    style="margin-left: 30px; margin-right: 30px;"
>

<p>Based on the estimated proportions of B.1.1.7, we provide the estimated expected absolute number of B.1.1.7 cases through time:</p>

<img
    src="https://raw.githubusercontent.com/cevo-public/Quantification-of-the-spread-of-a-SARS-CoV-2-variant/main/figures/g4.8/fig_cases_switzerland_joint.svg"
    style="max-width: 60%; margin-left: 30px; margin-right: 30px;"
>

<img
    src="https://raw.githubusercontent.com/cevo-public/Quantification-of-the-spread-of-a-SARS-CoV-2-variant/main/figures/g4.8/fig_cases_regions.svg"
    style="margin-left: 30px; margin-right: 30px;"
>

<p>On a weekly basis, we provide the estimated relative and absolute number of B.1.1.7 cases in an interactive plot:</p>

<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_b117_internal.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>


<h2>B.1.351 (or 501Y.V2) Variant</h2>
    
<p>We again provide the estimated relative and  absolute number of 501Y.V2 cases on a weekly basis in an interactive plot:</p>

<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_s501yv2_internal.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>


<h2>International Comparison</h2>
    
Finally we show the relative frequency of B.1.1.7 for Switzerland, Denmark and the UK in an interactive plot. The underlying data are GISAID genomes. In addition, for the UK, we use data on S gene target failure (S-dropout).
    
<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_b117_international.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>

<div class="source_info">
    Sources: Denmark: <a href="https://www.covid19genomics.dk/statistics">Danish Covid-19 Genome Consortium</a>;
    UK: <a href="https://www.gisaid.org/">GISAID</a>;
    UK S-dropout: <a href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">
        Technical briefings of Public Health England</a>;
    Switzerland: See acknowledgements belov
</div>


<h2>Acknowledgements:</h2>

<p>
    <b>Viollier:</b> Samples are provided by <a href="https://www.viollier.ch/">Viollier AG</a>. Whole genome sequenicing is performed
    through the <a href="https://bsse.ethz.ch/cevo/research/sars-cov-2/swiss-sequencing-consortium---viollier.html">
    Swiss SARS-CoV-2 Sequencing Consortium</a> lead by ETH ZÃ¼rich.
</p>
<p>
    <b>Risch:</b> Samples are provided and screened by <a href="https://www.risch.ch/">Labor Risch</a>. Genomic characterization is
    performed by Labor Risch, the <a href="https://www.unispital-basel.ch/ueber-uns/bereiche/medizin/kliniken-institute-abteilungen/infektiologie-spitalhygiene/ueber-uns/abteilungen/applied-microbiology-research-laboratory/">
    University Hospital Basel (Clinical Mircobiology)</a>
    and the University Hospitals of Geneva
    (<a href="https://www.unige.ch/medecine/demed/en/groupes-de-recherche/isabella-eckerle/">Group Eckerle</a> and
    <a href="https://www.unige.ch/medecine/demed/fr/groupes-de-recherche/668kaiser/">Group Kaiser</a>).
</p>
</body>
</html>
